tests/wpt/meta/import-maps/multiple-import-maps/already-resolved-dropped.html.ini

```diff
@@ -0,0 +1,2 @@
+[already-resolved-dropped.html]
+  expected: CRASH
```

Comment:
There are three crash tests and one failed test regression because of outdated module hook in script_module.
I opened a tracking issue #37553 for further improvement. These hooks seems to have similar large scope like resolving module specifiers.